apps/include/opt.h

```diff
@@ -311,27 +311,23 @@ typedef struct options_st {
     const char *helpstr;
 } OPTIONS;
 
-/*
- * A string/int pairing; widely use for option value lookup, hence the
- * name OPT_PAIR. But that name is misleading in s_cb.c, so we also use
- * the "generic" name STRINT_PAIR.
- */
+/* A string/int pairing; widely used for option value lookup. */
```

Comment:
I'd simply drop this commit. IMO the STRINT_PAIR name has some merit for the s_cb. (Perhaps it would be even better to name it `STR_INT_PAIR` though and use it everywhere even instead of OPT_PAIR as the OPT_PAIR name is not really descriptive.)

Reply:
All right, dropped.